packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -11,16 +11,13 @@ import https from 'https'
 import * as PromClient from 'prom-client'
 import { Counters, Histograms } from './common/metrics'
 import config, { getVersion } from './config'
-import { DomainAuthService } from './domain/auth/domainAuth.service'
-import { DomainService } from './domain/domain.service'
-import { DomainQuotaService } from './domain/quota/domainQuota.service'
 import { handleGetBlindedMessagePartialSig } from './signing/get-partial-signature'
 import { handleGetQuota } from './signing/query-quota'
 
 require('events').EventEmitter.defaultMaxListeners = 15
 
 export function createServer() {
-  const domainService = new DomainService(new DomainAuthService(), new DomainQuotaService())
+  // const domainService = new DomainService(new DomainAuthService(), new DomainQuotaService())
```

Comment:
why is this + the below `addMeteredEndpoint` lines commented out? Can they be deleted (or have a comment explaining why not)?